.changeset/tall-eyes-wait.md

```diff
@@ -0,0 +1,5 @@
+---
+"@tiptap/extension-drag-handle": major
```

Comment:
Since you are pushing this to main right now - are you sure this is a major change?

Reply:
Updated the changeset to more descriptive "Dismissable popup on keydown, when the popup is visible and the editor has focus, typing should hide the popup." does that work for you?